File: poo/1. Calculating an Employee's Salary/salary.php
<?php
abstract class salary {
    protected int $identifity;
    protected string $name;
    protected string $lastName;
    protected int $salario;

    public function __construct(int $identifity, string $name, string $lastName, int $salario) {
        $this->identifity = $identifity;
        $this->name = $name;
        $this->lastName = $lastName;
        $this->salario = $salario;
    }
    public function getIdentifity(): int {
        return $this->identifity;
    }
    public function getName(): string {
        return $this->name;
    }
    public function getLastName(): string {
        return $this->lastName;
    }
    public function getSalario(): int {
        return $this->salario;
    }

    abstract public function payTotal(): float;

    public function salarios(): void {
        if ($this->salario < (2 * 1423500)) {
            $this->salario = $this->salario +  200000;
        }else{
            $this->salario = $this->salario + 0;
        }
    }
}
?>

File: poo/1. Calculating an Employee's Salary JS/salary.java
public abstract class salary {
    private final int identifity;
    private final String name;
    private final String lastName;
    private int salario;

    public salary(int identifity, String name, String lastName, int salario){
        this.identifity = identifity;
        this.name = name;
        this.lastName = lastName;
        this.salario = salario;
    }
    public int getIdentifity(){
        return this.identifity;
    }
    public String getName(){
        return this.name;
    }
    public String getLastName(){
        return this.lastName;
    }
    public int getSalario(){
        return this.salario;
    }

    public abstract double payTotal();


    public void salarios(){
        if(salario < (2 * 1423500)){
            this.salario = this.salario + 200000;
        }else{
            this.salario = this.salario + 0;
        }
    }
}